auth0/reset-tenant.bash

```
@@ -0,0 +1,59 @@
+# Delete the API
+if [ ! -z $(curl -s -H "Authorization: Bearer $AT" "https://$TENANT/api/v2/resource-servers" | jq -r ".[] | select(.name == \"$API_NAME\") | .name") ]
+then
+  echo "Delete API $API_NAME ($API_URL)..."
+  apiID=$(curl -s -H "Authorization: Bearer $AT" "https://$TENANT/api/v2/resource-servers" | jq -r ".[] | select(.name == \"$API_NAME\") | .id")
+  curl -s -H "Authorization: Bearer $AT" -X DELETE https://$TENANT/api/v2/resource-servers/$apiID
+else
+  echo "API $API_NAME ($API_URL) already deleted"
+fi
+
+# Delete the user
+if [ ! -z $(curl -s -H "Authorization: Bearer $AT" --get --data-urlencode "email=$USER_EMAIL" https://$TENANT/api/v2/users-by-email | jq -r ".[] | select(.email == \"$USER_EMAIL\") | .email") ]
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)